__tests__/app.test.js

```diff
@@ -3,27 +3,7 @@ const setup = require('../data/setup');
 const request = require('supertest');
 const app = require('../lib/app');
 const Order = require('../lib/models/Order');
-
-// TODO: Remove this function & use the Order model
-async function createOrder({ product, quantity }) {
-  const { rows } = await pool.query(
-    'INSERT INTO orders(product, quantity) VALUES ($1, $2) RETURNING *;',
-    [product, quantity]
-  );
-  return new Order(rows[0]);
-}
-
-// TODO: Remove this function & use the Order model
-async function getOrderById(id) {
-  const { rows } = await pool.query(
-    'SELECT * FROM orders WHERE id=$1;',
-    [id]
-  );
-
-  if (!rows[0]) return null;
-
-  return new Order(rows[0]);
-}
+const { insert, getAll, getById } = require('../lib/models/Order');
```

Comment:
because you imported the Order model on line 5 you don't need to import each function individually here on line 6.